spec/calendar.html

```diff
@@ -329,6 +329,7 @@ <h1>FormatCalendarAnnotation ( _id_, _showCalendar_ )</h1>
     <emu-clause id="sec-temporal-calendarequals" aoid="CalendarEquals">
       <h1>CalendarEquals ( _one_, _two_ )</h1>
       <emu-alg>
+        1. If _one_ and _two_ are the same Object value, return *true*.
```

Comment:
this AO doesn't assert that its arguments are of type `Object` (which should be done in its description prose, which it's currently missing)